src/app/core/authentication/component/auth/auth.component.ts

```diff
@@ -1,4 +1,11 @@
 import { Component } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { faGoogle, faGithub, faFacebook } from '@fortawesome/free-brands-svg-icons';
+import { faArrowRightLong } from '@fortawesome/free-solid-svg-icons';
+import { AuthService } from '../../services/auth.service';
+import { SnackbarService } from '../../../../shared/services/snackbar/snackbar.service';
```

Comment:
Let's use absolute imports instead of relative imports for better readability and potentially easier refactoring